### ISSUE TITLE
If optimisation fails, original image is deleted

### ISSUE THREAD

**Issue by tonyastley** [ImageOptim/ImageOptim#442]
When optimising files, if the optimisation fails and the exclamation icon is displayed, the original image disappears completely.

**Comment by elena18** [comment]
Exactly: I have the same issue. However even f the optimization seems fine from the app, sometimes the file is deleted anyway

**Comment by RealBenAnderson** [comment]
Came and found this program here on github specifically because of this problem. It makes the application unusable because now, I have to check through literally hundreds of thousands of files to make sure none are missing, which is not practical. The workaround I am using is to create a copy of them all, then run the program, then merge it back with the original folder and just accept that some of the files will not be optimized.
